src/providers/krb5/krb5_child.c

```diff
@@ -121,6 +121,7 @@ struct krb5_req {
     struct cli_opts *cli_opts;
 };
 
+static bool krb5_child_has_setid_cap = true;
```

Comment:
Hi,

would it be possible to add `krb5_child_has_setid_cap` to `struct krb5_req` and avoid the global variable?

bye,
Sumit

Reply:
I used global variable because logically it is a property of a process, not of a backend's request.

But ok, I'll move it to 'krb5_req'.

Reply:
Done.